docs/collections.md

```diff
@@ -0,0 +1,138 @@
+# Collections
+Collections define the structure for the different content types on your static site. Since every site is different, the `collections` settings will be very different from one site to the next.
```

Comment:
Note that the examples below are present in the CMS repo example, and link to the relevant source.